src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/DprReportingIntegrationTest.kt

```diff
@@ -74,93 +79,118 @@ class DprReportingIntegrationTest : SqsIntegrationTestBase() {
+          .jsonPath("$[0].authorised").isEqualTo("true")
       }
-    }
-  }
-
-  @DisplayName("GET /definitions/incident-report/summary")
-  @Nested
-  inner class GetDefinitionDetails {
-    private val url = "/definitions/incident-report/summary"
-
-    @DisplayName("is secured")
-    @Nested
-    inner class Security {
-      @DisplayName("by role and scope")
-      @TestFactory
-      fun endpointRequiresAuthorisation() = endpointRequiresAuthorisation(
-        webTestClient.get().uri(url),
-        "INCIDENT_REPORTS__RO",
-      )
-    }
-
-    @DisplayName("works")
-    @Nested
-    inner class HappyPath {
 
       @Test
-      fun `returns the definition of the report`() {
+      fun `returns the not auth definitions of the reports`() {
+        manageUsersMockServer.stubLookupUsersRoles("request-user", listOf("ANOTHER_USER_ROLE"))
+
         webTestClient.get().uri(url)
-          .headers(setAuthorisation(roles = listOf("ROLE_INCIDENT_REPORTS__RO"), scopes = listOf("read")))
+          .headers(setAuthorisation(roles = listOf(systemRole), scopes = listOf("read")))
           .header("Content-Type", "application/json")
           .exchange()
           .expectStatus().isOk
           .expectBody()
-          .json(
-            """
-              {
-  "id": "incident-report",
-  "name": "Incident report summary",
-  "description": "List of all incidents filtered by dates, types, status and locations (INC0009)",
-  "variant": {
-    "id": "summary",
-    "name": "Incident Report Summary",
-    "resourceName": "reports/incident-report/summary",
-    "description": "List of all incidents filtered by dates, types, status and locations",
-    "printable": true
-  }
-}
-            """.trimIndent(),
-          )
+          .jsonPath("$.length()").isEqualTo(3)
+          .jsonPath("$[0].authorised").isEqualTo("false")
       }
     }
-  }
-
-  @DisplayName("GET /reports/incident-report/summary")
-  @Nested
-  inner class RunReportIncidentSummary {
-    private val url = "/reports/incident-report/summary"
 
-    @DisplayName("is secured")
+    @DisplayName("GET /definitions/incident-report/summary")
     @Nested
-    inner class Security {
-      @DisplayName("by role and scope")
-      @TestFactory
-      fun endpointRequiresAuthorisation() = endpointRequiresAuthorisation(
-        webTestClient.get().uri(url),
-        "INCIDENT_REPORTS__RO",
-      )
+    inner class GetDefinitionDetails {
+      private val url = "/definitions/incident-report/summary"
+
+      @DisplayName("is secured")
+      @Nested
+      inner class Security {
+        @DisplayName("by role and scope")
+        @TestFactory
+        fun endpointRequiresAuthorisation() = endpointRequiresAuthorisation(
+          webTestClient.get().uri(url),
+          systemRole,
+        )
+      }
+
+      @DisplayName("works")
+      @Nested
+      inner class HappyPath {
+
+        @Test
+        fun `report definition denied when user has incorrect role`() {
```

Comment:
would this test sit more naturally in the `Security` class above? that's about permissions checks whereas `HappyPath` is intended for successful requests

Reply:
Yes - will move